### ISSUE TITLE
NHML-1.8.1.5 not detecting AMD RX 480

### ISSUE THREAD

**Issue by spardue** [nicehash/NiceHashMiner-Archived#1339]
Hi,
Description is in title. Although, it looks like the logs are picking up an AMD RX 480 but not actually showing it in the list of devices to mine with.

I've attached the logs:
[log.txt](https://github.com/nicehash/NiceHashMiner/files/1480654/log.txt)

**Comment by spardue** [comment]
Here is part of the log that says it is picking up the RX 480:

[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] QueryCudaDevices END
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] QueryOpenCLDevices START
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] AMDOpenCLDeviceDetection found no devices. AMDOpenCLDeviceDetection returned: AMDOpenCLDeviceDetection::QueryDevices() exception: clGetDeviceIDs
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] QueryOpenCLDevices END
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] QueryAMD START
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] Checking AMD device (driver): Intel(R) HD Graphics 3000 (9.17.10.4459)
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] Checking AMD device (driver): NVIDIA GeForce GTX 970 (22.21.13.8205)
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] Checking AMD device (driver): Radeon (TM) RX 480 Graphics (22.19.662.4)
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] Driver version seems to be 21.19.164.1 or higher. NeoScrypt and Lyra2REv2 will be removed from list
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] QueryAMD END
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] Cuda NVIDIA/CUDA device count GOOD
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] AMD GPU device count GOOD
[2017-11-15 20:36:39] [INFO] [ComputeDeviceManager.Query] virtual memory size GOOD

**Comment by WickedWireZ** [comment]
Hey did you ever figure out what was wrong?
Because I'm having the same issues

**Comment by spardue** [comment]
No, I switched to mining ETH which I'm glad I did due to the hack.
